datacenter/dtr/2.1/guides/release-notes.md

```diff
@@ -13,6 +13,27 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](install/upgrade.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.1.5
+
+(16 March 2017)
+
+**Bug fixes**
+
+* Fix memory leaks in the DTR API and Registry containers. Severity: high
+* Fixed an issue causing RethinkDB to not start due to DNS errors when
+the RethinkDB containers were not restarted at the same time. Severity: high
+* The UI seemed to load indefinitely when viewing V1 manifests that didn't
+include a 'size' property. To fix this, manifests without this field are no
+longer displayed to the user. Severity: medium
+* Made the DTR installer more resilient when using an NFS storage backend. If
+you're using NFS as a storage backend, DTR will be reconfigured when upgrading.
```

Comment:
maybe: "If you're using NFS as a storage backend, DTR had a chance of hanging when reconfiguring or upgrading."